forms-web-app/.eslintrc

```diff
@@ -1,6 +1,6 @@
 {
   "parserOptions": {
-    "ecmaVersion": 6,
+    "ecmaVersion": 9,
```

Comment:
allows promises and `...` spreading in objects

Reply:
Booyah!